Inputting/InputDefinitions/TouchscreenInputDefinition.cs

```diff
@@ -1,25 +1,24 @@
-﻿namespace Inputting.InputDefinitions
+﻿using Inputting.Inputs;
+
+namespace Inputting.InputDefinitions
 {
     /// <summary>
-    /// A touchscreen input is an input in the form of <c>x,y</c>, e.g. <c>120,215</c>,
-    /// with <c>x</c> and <c>y</c> being within 0 (inclusive) and the specified max width/height (exclusive).
-    /// The resulting input's effective text will be "touch".
-    /// The touched coordinates will be passed via additional data in the form of a 2-tuple <c>(x, y)</c>.
+    /// This input definition creates inputs of type <see cref="TouchscreenInput"/>.
     /// </summary>
-    public struct TouchscreenInputDefinition : IInputDefinition
+    public readonly struct TouchscreenInputDefinition : IInputDefinition
     {
-        public const string EffectiveText = "touch";
-
+        private readonly string _touchscreenName;
         private readonly int _width;
         private readonly int _height;
 
-        public TouchscreenInputDefinition(int width, int height)
+        public TouchscreenInputDefinition(string touchscreenName, int width, int height)
         {
+            _touchscreenName = touchscreenName;
             _width = width;
             _height = height;
         }
 
-        private const string Number = @"(?:0|[1-9]\d{0,3})";
+        private const string Number = @"(?:[0-9]{1,4})";
```

Comment:
Feature request: have it limit the length of Number based on the width and height of the touch area

Reply:
is this okay if e.g. a 2-digit-wide touchscreen won't allow 3-digits with leading zeroes then?

Reply:
Yes